CHANGELOG.md

```diff
@@ -51,6 +51,7 @@
 * (wasm) [\#254](https://github.com/line/lbm-sdk/pull/254) Specify wasm event types
 * (x) [\#255](https://github.com/line/lbm-sdk/pull/255) Remove legacy from modules
 * (perf) [\#320](https:/github.com/line/lbm-sdk/pull/320) internal objects optimization (BaseAccount, Balance & Supply)
+* (proto) [\#344](https://github.com/line/lbm-sdk/pull/344) move SigBlockHeight from TxBody into AuthInfo
```

Comment:
How about changing the scope to `auth` rathe than `proto`?

Reply:
```suggestion
* (auth) [\#344](https://github.com/line/lbm-sdk/pull/344) move SigBlockHeight from TxBody into AuthInfo
```